src/components/Article.vue

```diff
@@ -33,7 +33,10 @@
                     styleClass="article-detail-content"
```

Comment:
Could you please also add this to the pull request?

```suggestion
                    class="article-detail-content"
```

Reply:
I did it myself  😄